assets/CSS/style.css

```diff
@@ -1,3 +1,667 @@
+  flex: none;
+  width: 544px;
+  height: 448px;
+  background-image: url(/assets/images/Snapshoot-Portfolio.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-radius: 8px;
+  justify-content: center;
+}
+
+.story-snapshoot-potofolio2 {
+  flex: none;
+  width: 544px;
+  height: 448px;
+  background-image: url(/assets/images/Snapshoot-portofolio-stories.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-radius: 8px;
+  justify-content: center;
+}
+
+.about-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  background-color: #fff;
+  background-image: none;
+  border-top-right-radius: 74px;
+  border-bottom-left-radius: 0;
+  margin-top: 114px;
+  width: 1440px;
+  height: auto;
+}
+
+.about-wrapper .text {
+  display: inline-block;
+  width: 35%;
+  height: 670px;
+  margin: 0;
+  margin-left: 24px;
+}
+
+.form-section p {
+  text-align: center;
+  padding-left: 12px;
+  width: 569px;
+  height: 102px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  margin-left: 460px;
+}
+
+.about-wrapper .text p {
+  width: auto;
+  font-family: "poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-left: 24px;
+  margin-top: 0;
+  margin-bottom: 0;
+  overflow: hidden;
+  flex: none;
+}
+
+.about-wrapper .text .social-media {
+  display: inline-block;
+  margin-left: 24px;
+  margin-top: 0;
+}
+
+.see-project-btn .get-resume-link {
+  width: 138px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  margin-top: 24px;
+  margin-left: 24px;
+}
+
+.list-container {
+  list-style-type: none;
+  font-size: 20px;
+  font-family: "Poppins", sans-serif;
+  font-weight: 500;
+  width: 50%;
+  margin-left: 150px;
+}
+
+.list-icon {
+  background-image: url(/assets/images/Vector.png);
+}
+
+.list-item {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  list-style-type: none;
+  display: flex;
+  padding-left: 0;
+  margin-left: 0;
+}
+
+.list-title img {
+  width: 12px;
+  height: 6px;
+  margin-left: 465px;
+}
+
+.list-item li img {
+  width: 48px;
+  height: 48px;
+  vertical-align: middle;
+}
+
+.list {
+  border-block-width: 1px;
+  border-bottom-style: inset;
+}
+
+.list .list-title h3 {
+  display: inline-block;
+}
+
+.list-title .vector {
+  width: 24px;
+  height: 24px;
+}
+
+.list-title .vector1 {
+  width: 24px;
+  height: 24px;
+  margin-left: 538px;
+}
+
+.form-section {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 846px;
+  background-color: #6070ff;
+  background-image: url(/assets/images/abc.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  border-top-left-radius: 73px;
+  text-align: center;
+}
+
```

Comment:
This section doesn't quite look like the one from the template design. Be mindful about borders, margins, padding, fonts among other things and review the Figma template to make it look as similar as possible.